# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/mp4/MP4Metadata.h
# Commit: da557d1806dd
# Full Hash: da557d1806dd9e1e519e00a4481b2e075f722ae8
# Author: Butkovits Atila <abutkovits@mozilla.com>
# Date: 2023-01-27 22:54 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing bustages complaining about AVIFDecodedData. CLOSED TREE
# ==============================================================================

diff -r d980bebc6d26 -r da557d1806dd dom/media/mp4/MP4Metadata.h
--- a/dom/media/mp4/MP4Metadata.h	Fri Jan 27 20:35:34 2023 +0000
+++ b/dom/media/mp4/MP4Metadata.h	Fri Jan 27 22:54:08 2023 +0200
@@ -9,12 +9,12 @@
 
 #include "mozilla/UniquePtr.h"
 #include "DecoderData.h"
+#include "Index.h"
 #include "MediaData.h"
 #include "MediaInfo.h"
 #include "MediaResult.h"
 #include "ByteStream.h"
 #include "mp4parse.h"
-#include "SampleIterator.h"
 
 namespace mozilla {
 
@@ -26,7 +26,7 @@
  public:
   size_t Length() const;
 
-  bool GetIndice(size_t aIndex, MP4SampleIndex::Indice& aIndice) const;
+  bool GetIndice(size_t aIndex, Index::Indice& aIndice) const;
 
   explicit IndiceWrapper(Mp4parseByteData& aRustIndice);
 